examples/templates/form.html

```diff
@@ -2,15 +2,13 @@
 {% from 'bootstrap/form.html' import render_form, render_field %}
 
 {% block content %}
-    <h1>render_form</h1>
+    <h1>Standard Form</h1>
     {{ render_form(form) }}
 
-    <h1>render_field</h1>
-    <form method="post">
-    {{ form.csrf_token }}
-    {{ render_field(form.username) }}
-    {{ render_field(form.password) }}
-    {{ render_field(form.remember) }}
-    {{ render_field(form.submit) }}
-    </form>
```

Comment:
Why delete this example? People sometimes need to render the field one by one.

Reply:
Yes, I'll move it into the seperate documentation file